R/pkg/R/DataFrame.R

```diff
@@ -78,6 +78,55 @@ dataFrame <- function(sdf, isCached = FALSE) {
 
 ############################ SparkDataFrame Methods ##############################################
 
+#' storageName
+#'
+#' Return a SparkDataFrame's name.
+#'
+#' @param x The SparkDataFrame whose name is returned.
+#' @family SparkDataFrame functions
+#' @rdname storageName
+#' @examples
+#'\dontrun{
+#' sparkR.session()
+#' path <- "path/to/file.json"
+#' df <- read.json(path)
+#' storageName(df)
```

Comment:
since they have the same `rdname` instead of two blocks of examples you should merge into one.
In that case it would make more sense to check `storageName` - here checking without setting it first doesn't seem to make a lot of sense?